app/(gcforms)/[locale]/(form filler)/id/[...props]/actions.ts

```diff
@@ -15,7 +15,7 @@ export async function submitForm(
   values: Responses,
   language: string,
   formRecordOrId: PublicFormRecord | string
-): Promise<{ id: string; submissionId?: string | false; error?: Error }> {
+): Promise<{ id: string; submissionId?: string; error?: Error }> {
```

Comment:
According to our recent findings this should not have any impact on the server action identifier (I was able to validate that locally once again)